src/lib/dataAccess.js

```diff
@@ -1,26 +1,38 @@
 import axios from "axios";
 
 const baseURL = process.env.REACT_APP_ADMIN_BACKEND_URL;
+axios.defaults.withCredentials = true;
+
+export function getCookie(name) {
+    const r = document.cookie.match("\\b" + name + "=([^;]*)\\b");
+    return r ? r[1] : undefined;
+}
 
 export const httpGet = (endPoint) => {
     const url = baseURL + endPoint;
+
     return axios
         .get(url, {
             headers: {
                 "Content-Type": "application/json",
             },
+            withCredentials: true,
         })
         .catch((error) => {
             throw error;
         });
 };
 
-export const httpPost = (endPoint, body) => {
+export const httpPost = (endPoint, body, isLogin = false) => {
     const url = baseURL + endPoint;
+    const headers = {};
+    if (!isLogin) {
+        headers["X-Xsrftoken"] = getCookie("_xsrf");
+    }
```

Comment:
This logic is needed since there is no XSRF token to set for an initial login POST request.